src/components/pages/ParentFamily/ParentFamilyHome.js

```diff
@@ -58,66 +56,89 @@ const ParentFamilyHome = props => {
   };
 
   return (
-    <>
-      <ParentSidebar />
-      <Content
-        className="family-page-container"
-        style={{
-          backgroundImage: `url(${cloudbg})`,
-        }} //background image here while troubleshooting LESS rendering issue
+    <Content
+      className="family-page-container"
+      style={{
+        backgroundImage: `url(${cloudbg})`,
+      }} //background image here while troubleshooting LESS rendering issue
+    >
+      <Modal
+        title="Add Student"
+        visible={addStudentVisible}
+        onOk={handleAddStudentOk}
+        confirmLoading={addStudentConfirmLoading}
+        onCancel={handleAddStudentCancel}
+        footer={null}
       >
-        <Banner />
-        <Modal
-          title="Add Student"
-          visible={addStudentVisible}
-          onOk={handleAddStudentOk}
-          confirmLoading={addStudentConfirmLoading}
-          onCancel={handleAddStudentCancel}
-          footer={null}
-        >
-          <CreateNewStudent />
-        </Modal>
+        <CreateNewStudent />
+      </Modal>
 
-        <Row className="family-cards">
-          <Col span={8}>
-            <Card style={{ width: 300 }} className="parent-card">
-              <Meta
-                avatar={<Avatar src={user.avatarUrl} />}
-                title={user.name}
+      <Row className="family-cards">
+        <Col span={8}>
+          <Card className="parent-card">
+            <div className="card-info">
+              {/* // make dynamic with state management */}
+              <Avatar
+                className="avatar"
+                src="https://joeschmoe.io/api/v1/random"
               />
-              <Button className="parent-view-account-button">
-                VIEW ACCOUNT
+              <Text className="card-name">Parent Name</Text>
+              <Button
+                className="parent-view-account-button parent-card"
+                onClick={() => history.push('/parent')}
+              >
+                View Account
+
               </Button>
               <Button
                 className="add-student-button"
                 onClick={showAddStudentModal}
               >
-                ADD STUDENT
+                Add Student
+              </Button>
+            </div>
+          </Card>
+        </Col>
+
+        <Col span={8}>
+          <Card className="student-card">
+            <div className="card-info">
+              {/* make dynamic with state management */}
```

Comment:
I don't think this is a life saver comment, but perhaps Paul has a different opinion. And even if so, I would just have this comment written out once in this file, not inside of each 'card-info' div.